src/labels/CustomLabels.labels

```diff
@@ -8727,6 +8735,30 @@ To check your existing settings, go to Duplicate Rules under Setup. For any rule
         <shortDescription>stgHelpUseDatedConvRates</shortDescription>
         <value>When checked, Dated Exchange Rates are used for currency conversion. If unchecked, the standard exchange rate is used even if Advanced Currency Management is enabled.</value>
     </labels>
+    <labels>
+        <fullName>stgInstallmentOppDisableAll</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Disable Installments Auto-Creation process</shortDescription>
+        <value>Disable All Installments</value>
+    </labels>
+    <labels>
+        <fullName>stgInstallmentOppDisableFirst</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Disable First Installment Auto-Creation on Create</shortDescription>
+        <value>Disable First Installment on Create</value>
+    </labels>
+    <labels>
+        <fullName>stgInstallmentOppEnableAll</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Always Create Next Installment</shortDescription>
+        <value>Always Create Next Installment</value>
```

Comment:
@sam-knox I re-highlight the parts that needs to be review. These are the short description for the custom labels. Also the text line 8390.